src/react/pages/Home.tsx

```diff
@@ -150,13 +154,15 @@ const Home = () => {
 
   return (
     <StyledMuiContainer>
-      <Network><Button color="primary" onClick={handleOpenNetworkModal}>{networkSelected}</Button></Network>
-      <Modal
-        open={showNetworkModal}
-        onClose={handleCloseNetworkModal}
-      >
-        <NetworkPicker handleCloseNetworkModal={handleCloseNetworkModal} setNetworkSelected={setNetworkSelected} networkSelected={networkSelected}></NetworkPicker>
-      </Modal>
+          <Network>
```

Comment:
Can you fix the indentation here to match the rest of the project?  Indentation 2 spaces, and try and match the opening and closing tags on the same level.  Thanks!

Reply:
Made a new commit with the correction. Hope I did this right.